src/app-layout/visual-refresh-toolbar/toolbar/index.tsx

```diff
@@ -106,8 +108,11 @@ export function AppLayoutToolbarImplementation({ appLayoutInternals }: AppLayout
             />
           </nav>
         )}
-        {breadcrumbs && (
-          <div className={clsx(styles['universal-toolbar-breadcrumbs'], testutilStyles.breadcrumbs)}>{breadcrumbs}</div>
+        {(breadcrumbs || discoveredBreadcrumbs) && (
+          <div className={clsx(styles['universal-toolbar-breadcrumbs'], testutilStyles.breadcrumbs)}>
+            {breadcrumbs}
+            {discoveredBreadcrumbs && <InternalBreadcrumbGroup {...discoveredBreadcrumbs} />}
+          </div>
```

Comment:
The `breadcrumbs` value here is also deduplicated into `discoveredBreadcrumbs` same way as any other instance on the page. No special treatment for "own breadcrumbs"